app/src/main/java/com/app/calculator/Operator.java

```diff
@@ -2,7 +2,12 @@
 
 public enum Operator {
     DEF("def","Select Operation"),
-    ADD("add","+");
+    ADD("add","+"),
+    MUL("multiplicate","*"),
```

Comment:
please add better naming, this should be MULTIPLICATION, don't be afraid of having long but easy to read variable / const names